src/style.css

```diff
@@ -1 +1,67 @@
 /* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
+}
+
+html {
+  max-width: 1200px;
+  min-width: 1024px;
```

Comment:
do not limit page width

Reply:
i have a task to make it adaptive:
- pay attention the mock is adaptive. Develop the layout to fit on 1024px and 1200px the same as on the mock.

that's why i limit it